### ISSUE TITLE
Bloom PostProcessing example, error thrown when light is added

### ISSUE THREAD

**Issue by MAG-AdrianMeredith** [playcanvas/engine#5844]
The new "renderpass" post processing example throws an exception when a shadow casting light is added.

### Steps to Reproduce

1. go to https://playcanvas.github.io/#/graphics/postprocessing
2. add `boardEntity.addComponent('light', {type:'directional', castShadows: true});` to line 88
3.playcanvas.js:29104 Uncaught TypeError: Cannot read properties of null (reading 'width') is found

It appears the shadowCaster camera has no render target...
<img width="1256" alt="image" src="https://github.com/playcanvas/engine/assets/100127317/9fc53aee-4a70-4b31-828b-8a1c1b37b4e7">

**Comment by mvaligursky** [comment]
Yep, this is in preview and many things are not done. Specially, directional shadow passes and depth/color passes are not in a form to be easily added to the camera passes, and so those don't work. This should be ready for the next minor engine release (working on some of it now)

**Comment by MAG-AdrianMeredith** [comment]
No worries, its looking great just thought I'd be good to get it logged in case you weren't aware
